src/DatePicker/DatePicker.js

```diff
@@ -206,13 +209,55 @@ class DatePicker extends Component {
         date: date,
       });
     }
+
+    setTimeout(() => {
```

Comment:
We can't keep that timeout like that. What's the need?
At least, we need to clear it when the component unmount.